### PR TITLE
Update readme and Present Code

### PR DESCRIPTION
**Pull request by AbayomiOlaoye**
## Intro
This project marks my first and cherished capstone project here at Microverse.
In this PR, I present to you my work for review and constructive feedback regarding the level of my commitment and expertise exuded by this piece.

## Links
- This is a link to the deployed version of this project. Click [here](https://abayomiolaoye.github.io/Youth-In-Tech-Project/)
- Here is to the recorded video unveiling the project. You can check [here](https://www.loom.com/share/79097e4bbd2e4cbcb02393282aa9eee3)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,4 +53,4 @@ Give a ⭐️ if you like this project from a beginner like me!
```
Comment (cynthiainga) [inline README.md:55]: - [x] The `README` should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/), please kindly add it 😉 😉

File: README.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 # Youth-In-Tech-Project
 Here's my first capstone project, featuring a young and dynamic group of youngsters in tech. The project eas inspired by my crave to see young minds being inspired by their colleagues who have seized the opportunity to learn the language of the future - not only for the security it affords but the feeling of accomplishment it bestows. I really hope to see my fellow country young men to dare and dive into this atmosphere of CHANGE for a better World. 
 
-## Live video link
-https://vimeo.com/726372669/3dd9de63e0
+## Live Presentation Video Link
+https://www.loom.com/share/df9b50bd237d4c51948291ffd7a39161
```
Comment (cynthiainga) [inline README.md]: - [ ] Please as required the maximum length of the video is 5 minutes, I enjoyed watching your video but is too long 🙃 😉, kindly record a video with a max length of 5 minutes and also the video should contain a highlight about a piece of the code that you finds interesting or proud of making 😊 😁.

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @AbayomiOlaoye, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] Nice design 👍 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -36,21 +36,23 @@ Open your code editor, open the folder (cloned repo) and play around with the co
...
 
 - Special thanks to @microverse for this opportunity
 - A worthy acknowledgment of my colleagues who presently serve as code reviewers in the institute for their constructive feedback.
 
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
\ No newline at end of file
+This project is [MIT](./MIT.md) licensed.
```
Comment (medaminedev66) [inline README.md:58]: [OPTIONAL] This MIT link is broken. I recommend fixing this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

**Review by medaminedev66 — APPROVED** [review]
Hi @AbayomiOlaoye 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💯 
Congratulations! 🎉 

### Highlights
- Your repo has a descriptive Readme.
- Your code organization is really good.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (AbayomiOlaoye) [reply README.md:58]: Thanks for the suggestion. It's been effected.

**Pull request merged**